mobile/src/components/uam-groups-config/uam-configure-groups.tsx

```diff
@@ -0,0 +1,165 @@
+import React, { FC, useEffect } from 'react';
+import { ScrollView, View } from 'react-native';
+
+import { AppScreenName, PaginationDefaultValue } from '~/common/enums/enums';
+import { GroupsCreateRequestDto } from '~/common/types/types';
+import { Button, Input, Text } from '~/components/common/common';
+import {
+  PermissionsTable,
+  UsersTable,
+} from '~/components/uam-groups-config/components/components';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppNavigate,
+  useAppSelector,
+  usePagination,
+  useSelectedItems,
+} from '~/hooks/hooks';
+import { groupsCreationActions, uamGroupEditActions } from '~/store/actions';
+import { groupCreateClient } from '~/validation-schemas/validation-schemas';
+
+import { CREATE_GROUP_DEFAULT_PAYLOAD } from './common/constants/create-group-default.constants';
+import { styles } from './styles';
+
+const UAMGroupsConfigure: FC = () => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
+  const { group } = useAppSelector((state) => state.uamGroupEdit);
+
+  const { users, permissions } = useAppSelector(
+    (state) => state.uamGroupCreation,
+  );
+
+  const { page: usersPage, handlePageChange: handleUserPageChange } =
+    usePagination();
+
+  const {
+    page: permissionsPage,
+    handlePageChange: handlePermissionsPageChange,
+  } = usePagination();
+
+  const { control, handleSubmit, errors, reset } =
+    useAppForm<GroupsCreateRequestDto>({
+      defaultValues: CREATE_GROUP_DEFAULT_PAYLOAD,
+      validationSchema: groupCreateClient,
+    });
+
+  const {
+    items: permissionIds,
+    handleToggle: handleTogglePermissions,
+    setItems: setDefaultPermissionIds,
+  } = useSelectedItems<number>(group?.permissionIds ?? []);
+
+  const {
+    items: userIds,
+    handleToggle: handleToggleUsers,
+    setItems: setDefaultUserIds,
+  } = useSelectedItems<number>(group?.userIds ?? []);
+
+  const paginationForUsersTable = {
+    page: usersPage,
+    setPage: handleUserPageChange,
+  };
+
+  const paginationForPermissionsTable = {
+    page: permissionsPage,
+    setPage: handlePermissionsPageChange,
+  };
+
+  const handleCreateGroup = async (): Promise<void> => {
+    await dispatch(
+      groupsCreationActions.createGroup({
+        name: control._formValues.name,
+        permissionIds,
+        userIds,
+      }),
+    ).unwrap();
+    navigation.navigate(AppScreenName.UAM);
+  };
+
+  const handleEditGroup = async (): Promise<void> => {
+    if (!group) {
+      return;
+    }
+
+    await dispatch(
+      uamGroupEditActions.editGroup({
+        id: group.id,
+        payload: {
+          name: control._formValues.name,
```

Comment:
Don't do so, create a constant for field name if needed
```suggestion
          name: control._formValues.name,
```

Reply:
Isn't fixed